sriov_network_operator/README.md

```diff
@@ -0,0 +1,3 @@
+# SRIOV NETWORK OPERATOR project
```

Comment:
Please add more details about the scenario tested and the passed parameters

Reply:
Done